source/crud/tests/unified/bulkWrite-update-validation.yml

```diff
@@ -1,140 +1,73 @@
-# This file was created automatically using mongodb-spec-converter.
-# Please review the generated file, then remove this notice.
+description: "bulkWrite-update-validation"
+
+schemaVersion: "1.0"
```

Comment:
I'm not sure why the converted CRUD v2 tests all used "1.1" (perhaps it was just in the template), but these validate against the original schema so I've lowered the version. To quote the unified test format spec:

> Test files SHOULD be conservative when specifying a schema version. For example, if the latest schema version is 1.1 but the test file complies with schema version 1.0, the test file should specify 1.0.

I came across this after restoring the older schemas for [DRIVERS-1754](https://jira.mongodb.org/browse/DRIVERS-1754).

Reply:
That is correct, I used schemaVersion 1.1 in those tests. I recall there being a reason for this but can't remember, so if 1.0 works as well that's fine

Reply:
Will be handled by https://github.com/mongodb/specifications/pull/1000 for [DRIVERS-1775](https://jira.mongodb.org/browse/DRIVERS-1775).